### PR TITLE
Ember 3.26 Release

### PR DESCRIPTION
**Pull request by jaredgalanis**
### ember-source

- [x] [#19255](https://github.com/emberjs/ember.js/pull/19255) [DEPRECATION] Deprecate transition methods of controller and route per [RFC #674](https://github.com/emberjs/rfcs/blob/master/text/0674-deprecate-transition-methods-of-controller-and-route.md).
- [x] [#19345](https://github.com/emberjs/ember.js/pull/19345) [DEPRECATION] Deprecate `<LinkTo>` positional arguments per [RFC #698](https://github.com/emberjs/rfcs/blob/master/text/0698-deprecate-link-to-positional-arguments.md).
- [x] [#19346](https://github.com/emberjs/ember.js/pull/19346) [DEPRECATION] Deprecate `{{#with}}` per [RFC #445](https://github.com/emberjs/rfcs/blob/master/text/0445-deprecate-with.md)
- [x] [#19358](https://github.com/emberjs/ember.js/pull/19358) [DEPRECATION] Deprecate implicit injections per [RFC #680](https://github.com/emberjs/rfcs/blob/master/text/0680-implicit-injection-deprecation.md)
- [x] [#19359](https://github.com/emberjs/ember.js/pull/19359) [DEPRECATION] Deprecates old browser support policy per [RFC #685 New Browser Support Policy](https://github.com/emberjs/rfcs/blob/master/text/0685-new-browser-support-policy.md).
- [x] [#19371](https://github.com/emberjs/ember.js/pull/19371) [DEPRECATION] Deprecate implicit `this` property lookup fallback per [RFC #308](https://github.com/emberjs/rfcs/blob/master/text/0308-deprecate-property-lookup-fallback.md)
- [x] [#19372](https://github.com/emberjs/ember.js/pull/19372) [DEPRECATION] Adds deprecations for Classic edition and optional features per [RFC #704](https://github.com/emberjs/rfcs/blob/master/text/0704-deprecate-octane-optional-features.md) and [RFC #705](https://github.com/emberjs/rfcs/blob/master/text/0705-deprecate-jquery-optional-feature.md).
- [x] [#19373](https://github.com/emberjs/ember.js/pull/19373) [DEPRECATION] Deprecate old manager capabilities per [RFC #686](https://github.com/emberjs/rfcs/blob/master/text/0686-deprecate-old-manager-capabilities-versions.md)
- [x] [#19374](https://github.com/emberjs/ember.js/pull/19374) [DEPRECATION] Deprecate `hasBlock` and `hasBlockParams` per [RFC #689](https://github.com/emberjs/rfcs/blob/master/text/0689-deprecate-has-block.md).
- [x] [#19375](https://github.com/emberjs/ember.js/pull/19375) [DEPRECATION] Deprecate old class binding syntax and {{attrs}} per [RFC #691](https://github.com/emberjs/rfcs/blob/master/text/0691-deprecate-class-binding-and-class-name-bindings.md) and [RFC #690](https://github.com/emberjs/rfcs/blob/master/text/0690-deprecate-attrs-in-templates.md).
- [x] [#19381](https://github.com/emberjs/ember.js/pull/19381) [DEPRECATION] Deprecate Array Observers per [RFC #692](https://github.com/emberjs/rfcs/blob/master/text/0692-deprecate-array-observers.md).
- [ ] ~[#19379](https://github.com/emberjs/ember.js/pull/19379) [CLEANUP] Refactor DataAdapter to not use observers or array observers~
- [ ] ~[#19378](https://github.com/emberjs/ember.js/pull/19378) [BUGFIX] Fix typo in template-only-glimmer-components feature detection~
- [ ] ~[#19298](https://github.com/emberjs/ember.js/pull/19298) [BUGFIX] Route serialize did not extract param off proxy~
- [x] [#19469](https://github.com/emberjs/ember.js/pull/19469) [BUGFIX] Prevent eager argument consumption on modifier destruction
- [x] [#19405](https://github.com/emberjs/ember.js/pull/19405) [BUGFIX] Avoid instantiation errors when `app/router.js` injects the router service.
- [x] [#19436](https://github.com/emberjs/ember.js/pull/19436) [BUGFIX] Support observer keys with colons

### ember-data

- [ ] ~~[#7435](https://github.com/emberjs/data/pull/7435) [Test]: fix for DataWrapper refactor in emberjs (#7435)~~
- [ ] ~~[#7416](https://github.com/emberjs/data/pull/7416) [Deps]: bump ember packages (#7416)~~
- [ ] ~~[#7358](https://github.com/emberjs/data/pull/7358) [Chore]: TypeScript adapters (#7358)~~

### ember-cli

- [x] [#9473](https://github.com/ember-cli/ember-cli/pull/9473) Issue a better error message for add-on's missing an entry point (e.g. invalid `ember-addon.main` path)
- [x] [#9437](https://github.com/ember-cli/ember-cli/pull/9437) Add Prettier files to ".npmignore" file in addon blueprint
- [x] [#9436](https://github.com/ember-cli/ember-cli/pull/9436) Enable Embroider test scenario for addons
- [ ] ~~[#9435](https://github.com/ember-cli/ember-cli/pull/9435) Use "lint:fix" script in app and addon README files~~ 
- [ ] ~~[#9451](https://github.com/ember-cli/ember-cli/pull/9451) update blueprint deps~~

### CODE REVIEW

File: content/ember-3-26-released.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+2. You can pass to an observer a dependent key whose name includes a colon. ([#19343](https://github.com/emberjs/ember.js/issues/19343))
+3. The Glimmer VM has been updated to prevent eagerly consuming arguments during modifier destruction. ([#19469](https://github.com/emberjs/ember.js/pull/19469))
+
+#### Features
+
+Ember.js 3.26 introduced 0 features.
+
+#### Deprecations
+
+Ember.js 3.26 introduced 11 deprecations.
```
Comment (ijlee2) [inline content/ember-3-26-released.md:48]: TODO:

Go through the deprecation issues and see which ones we want to bring attention to.

**Comment by ijlee2** [comment]
I've finished with `ember-source`

**Comment by jaredgalanis** [comment]
> I've finished with `ember-source`

I'm done with CLI and working on Data

File: content/ember-3-26-released.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+1. Property fallback for implicit `this` has been deprecated. Please review the templates in your app and write `this.` when it is appropriate, e.g. change `{{localProperty}}` to `{{this.localProperty}}`. ([Deprecations Guide](https://deprecations.emberjs.com/v3.x#toc_this-property-fallback), [#19371](https://github.com/emberjs/ember.js/pull/19371))
+1. **Ember Classic has been deprecated.** ([Deprecations Guide](https://deprecations.emberjs.com/v3.x#toc_editions-classic), [#19372](https://github.com/emberjs/ember.js/pull/19372))
+
+    In preparation for v4.0 release, developers are encouraged to update their app to Ember Octane by following these steps:
+
+    - In `config/optional-features.json`, update the feature flags for Octane.
+
+        ```json
+        {
+          "application-template-wrapper": false,
```
Comment (ijlee2) [inline content/ember-3-26-released.md:64]:
```suggestion
          "application-template-wrapper": false,
          "jquery-integration": false,
```

**Comment by jaredgalanis** [comment]
Done reviewing Data, and I don't think there's anything to mention this time around.

**Review by locks — APPROVED** [review]

**Pull request merged**